javascript/extensible-search-suggestions.js

```diff
@@ -1,35 +0,0 @@
-;(function($) {
```

Comment:
I second this, can't hurt to keep this in place for people who wish to use it, or for people who are *already* using it.